src/home/views/index.tsx

```diff
@@ -8,57 +9,64 @@ import { orderListUrl } from "../../orders/urls";
 import { productListUrl, productVariantEditUrl } from "../../products/urls";
 import { OrderStatusFilter, StockAvailability } from "../../types/globalTypes";
 import HomePage from "../components/HomePage";
-import { HomePageQuery } from "../queries";
+import { useHomePage } from "../queries";
 
 const HomeSection = () => {
   const navigate = useNavigator();
   const { user } = useUser();
   const { channel } = useAppChannel();
 
+  const noChannel = !channel && typeof channel !== "undefined";
```

Comment:
I'd rather see this as a positive const, such as `channelExists` or `hasChannel`, especially since most places we use it, it's negated. Besides that it seems easier to overall think of positives +- negation, rather than negatives +- negation (that more easily created negated negative), don't you think?

Reply:
I left `noChannel` after the refactor in which I added default props in `HomePage`. In this component it's mostly positive value. What do you think?